tracing-subscriber/src/subscribe.rs

```diff
@@ -542,8 +542,8 @@ pub trait CollectExt: Collect + crate::sealed::Sealed {
 /// [collector]: tracing_core::Collect
 /// [stored data]: super::registry::SpanRef
 #[derive(Debug)]
-pub struct Context<'a, S> {
-    subscriber: Option<&'a S>,
+pub struct Context<'a, C> {
+    subscriber: Option<&'a C>,
```

Comment:
should probably rename this field to `collector` as well, but that's a slightly larger change. Feel free to make it in this branch or later.